docs/configuration.md

```diff
@@ -280,9 +283,9 @@ in the routing tree to mute/activate particular routes for particular times of t
 
 ```yaml
 name: <string>
-time_intervals:
-  [ - <time_interval_spec> ... ]
+time_intervals: [- <time_interval_spec> ...]
```

Comment:
Unnecessary reformatting.
Not going to point out the rest, make sure to remove them.